python/pip-sse3-gentoo2023.conf

```diff
@@ -0,0 +1,17 @@
+[global]
```

Comment:
Do we still need this config in 2023, ie will we still support `sse3`?

Reply:
we won't be supporting `avx` either, correct ?

Reply:
I deleted both the `sse3` and `avx` versions for `gentoo2023`

Reply:
My first thought is against renaming `avx{2,512}` to `x86-64-v{3,4}` because `avx{2,512}` will keep lingering in other places, such as `arch/avx{2,512}`, the wiki, the github repositores, `RSNT_ARCH`, etc, and changing them would require doing an inventory to change them everywhere. They are also more complex for a user to type (in places where they are exposed to users).

Reply:
Please don't rename those.
 
IMO the names don't really tell what they are about and they are more coarse grained.
e.g. `AVX` is mentioned as a requirement for `x86-64-v4` but so is `AVX512` hence a CPU that has the first and not the latter would be considered `v2` and therefore lumped in with   those that we call `SSE3`.

I think we would navigate us into a corner by restricting us to this `x86-64-v{3,4}` nomenclature.

Reply:
> Please don't rename those.
> 
> IMO the names don't really tell what they are about and they are more coarse grained. e.g. `AVX` is mentioned as a requirement for `x86-64-v4` but so is `AVX512` hence a CPU that has the first and not the latter would be considered `v2` and therefore lumped in with those that we call `SSE3`.
> 
> I think we would navigate us into a corner by restricting us to this `x86-64-v{3,4}` nomenclature.

`AVX` would actually fall in `v3`, not `v2`. CPUs without the support for any `avx` (i.e. `v2`) won't be supported in the new stack.

Reply:
> Please don't rename those.
> 
> IMO the names don't really tell what they are about and they are more coarse grained. e.g. `AVX` is mentioned as a requirement for `x86-64-v4` but so is `AVX512` hence a CPU that has the first and not the latter would be considered `v2` and therefore lumped in with those that we call `SSE3`.
> 
> I think we would navigate us into a corner by restricting us to this `x86-64-v{3,4}` nomenclature.

CPUs without at least `avx2` will not be supported in the 2023 environment.